pom.xml

```diff
@@ -1001,11 +1004,26 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <executions>
           <execution>
-            <id>run-integration-tests</id>
+            <id>run-minicluster-integration-tests</id>
             <goals>
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <skip>${accumulo.skipMiniClusterInFailsafe}</skip>
+              <groups>${accumulo.miniclusterTests}</groups>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
If a user overrides the categories by setting the documented surefire/failsafe properties, `groups` and `excludedGroups`, it will completely override all of the settings you've added to these executions. They will get two executions of the same tests, taking twice as long, as the second execution will clobber the reports from the first, before the verify phase.

I can try to explain more later, if this isn't clear.